force-app/main/default/flows/NKS_Chat_Journal_Thread_v_2.flow-meta.xml

```diff
@@ -15,14 +16,14 @@
             <name>Is_Authenticated</name>
             <conditionLogic>and</conditionLogic>
             <conditions>
-                <leftValueReference>Get_Chat_Transcript.CRM_Authentication_Status__c</leftValueReference>
+                <leftValueReference>Get_Messaging_Session.CRM_Authentication_Status__c</leftValueReference>
                 <operator>EqualTo</operator>
                 <rightValue>
                     <stringValue>Completed</stringValue>
                 </rightValue>
             </conditions>
             <conditions>
-                <leftValueReference>Get_Chat_Transcript.AccountId</leftValueReference>
+                <leftValueReference>Get_Messaging_Session.EndUserAccountId</leftValueReference>
```

Comment:
Litt usikker hvilket felt blir riktig å bruke for account; hvis AccountId er satt for EndMessagingUser er dette feltet også oppfylt